ballerina/Package.md

```diff
@@ -0,0 +1 @@
+This repository is for Including ballerina service management tools for publishing service data to WSO2 API manager service catalogs.
```

Comment:
Shall we add a brief explanation about the package by including some samples and exposed APIs.

Reply:
Updated, Please check